libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core
```

Comment:
From reading [Namespace guidance for the options pattern](https://docs.microsoft.com/en-us/dotnet/core/extensions/options-library-authors#namespace-guidance), it looks like the dotnetframework uses a single namespace across all its libraries to unify ServiceCollecitonExensions so the method show up just by adding a new package reference. 
How about establishing this for the BF packages and use the `Microsoft.Bot.Builder.Extensions.DependencyInjection;` namespace here?

Reply:
I've updated the latest to include this.